README.md

```diff
@@ -1 +1,8 @@
-# Norman Objects Package
\ No newline at end of file
+# Norman Objects Overview
+
+Norman Objects is a shared library that defines the core request and response models used across the Norman SDK and internal services.
+
+The library includes enums, data classes, and other structured objects that ensure consistency, type safety and clear contracts between services and SDK consumers.
```

Comment:
The library includes classes, enums, and other structured objects that establish clear, consistent, type-safe contracts between backend services and SDK clients.